content/how-to/account-management/using-RBAC-with-MyRackspace/index.md

```diff
@@ -0,0 +1,179 @@
+
+#### Assign By Products
+This method enables you to manage user access to an individual product or product group.
+
+To assign permissions by product or product group, perform the following steps:
+
+**Step 1.** Select **Assign By Product**.
+
+<img width="467" alt="Select assign by product" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC11.png">
+
+**Step 2.** Make a selection from the **Product or Product Group** column, then below screen appears.
+
+<img width="915" alt="Select from product or product group" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC12.png">
+
+**Step 3.** Update the user, user group, or Group membership permission.
+
+**Step 4.** Select **Save** Changes.
+
+
+#### Manage Group
+The Manage Groups tab enables you to create new User and Product groups and update members.
+
+**Create User Groups**
+
+**Step 1.** Select **Manage Groups** from top navigation bar.
+
+<img width="477" alt="Select manage group from top navigation" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC13.png">
+
+**Step 2.** At the top of the User Groups column, enter a name for the new group and select the green plus sign as shown in below image.
+
+<img width="477" alt="Enter name of the new group" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC14.png">
+
+**Step 3.** Select the **Group Members** to add and select Save Changes.
+
+**Create Product Group**
+
+**Step 1.** Select Manage Groups
+
+**Step 2.** At the top of the User Product column, enter a name for the new group and select the green plus sign as shown in below image.
+
+<img width="452" alt="myrack-account-RBAC15" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC15.png">
+
+**Step 3.** Select the Product to add and select Save Changes.
+
+**Grant Access to a User Group or Product Group**
+
+Use the following steps to grant access to a user group or product group:
+
+**Step 1.** Select Manage Groups.
+
+**Step 2.** Select an existing group from the User Groups or Product Groups column.
+
+<img width="470" alt="Select an existing group" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC16.png">
+
+**Step 3.** Select the member to add.
+
+<img width="404" alt="Select the member" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC17.png">
+
+**Step 4.** Select **Save** Changes.
+
+
+**Delete a User or Product Group**
+Use the following steps to delete a user or product group:
+
+**Step 1.** Select **Manage Groups** from top navigation bar.
+
+**Step 2.** Select an existing group from the User Groups or Product Groups column.
+
+<img width="908" alt="Select an existing user groups" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC18.png">
+
+**Step 3.** Select the **Actions** dropdown in the upper-right corner as shown in above image.
```

Comment:
are we supposed to be doing `dropdown` or `drop-down`?

Reply:
Change to drop-down